### PR TITLE
Add generator for Heroku deploy setup

### PR DESCRIPTION
**Pull request by kylesmile**
Generator for basic Heroku deploy setup, as part of an effort to create instructions for setting up a Heroku server for a project. Since I only referenced a few LightningCAD projects, this may be incomplete.

### CODE REVIEW

File: lib/generators/rolemodel/heroku/templates/Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -C config/puma.rb
+release: bin/heroku_release
```
Comment (calebwoods) [inline lib/generators/rolemodel/heroku/templates/Procfile:2]: I've seen most of our rails projects just have `bundle exec rails db:migrate` what else is the script getting us?

Comment (calebwoods) [reply lib/generators/rolemodel/heroku/templates/Procfile:2]: It looks the script handles the DB not existing, what's the way that happens?

Comment (kylesmile) [reply lib/generators/rolemodel/heroku/templates/Procfile:2]: `db:migrate` is probably sufficient. I think that script was adapted from something that was meant to decide whether it should run `schema:load` or `db:migrate`, but the consensus was that doing that was a bad idea. `app.json` covers running `schema:load` when creating a new server, anyway.

Comment (kylesmile) [reply lib/generators/rolemodel/heroku/templates/Procfile]: Removed the script in favor of calling `db:migrate` directly.

**Pull request merged**